src/org/springfield/rafael/mediafragment/server/MediaFragmentServer.java

```diff
@@ -173,11 +173,20 @@ public void handleGet() {
 					if (start == 0l && end > 1l) {
 						//only allowed with ticket allows
 						String wowzaUri = conf.getProperty("wowza-server-uri");
+						String apiKey = conf.getProperty("api-key");
 	
 						//get ticket
 						StringRepresentation entity = new StringRepresentation("<fsxml><properties><uri>"+fileIdentifier+"</uri></properties></fsxml>");
 						entity.setMediaType(MediaType.TEXT_XML);
 						Request request = new Request(Method.PUT, wowzaUri+"/acl/ticketaccess/"+ticket, entity);
+						if (apiKey != null && !apiKey.equals("")) {
```

Comment:
triple code duplication in this diff... And it looks like `dans-springfield-carl` has the same code twice as well. Would be nice if you could reuse the same piece of code in all five instances!